GOVERNANCE.md

```diff
@@ -107,7 +107,7 @@ The responsibilities of the Technical Committee are
 * coordinating event participation and communications regarding the Eiffel Community Open Source Projects
 
 Eiffel Community aims to operate as a consensus based community, if any decision requires a vote to move the Eiffel Community Open Source Projects forward, the members of the Technical Committee will vote on a one vote per member basis.
-Simple majority is required for any decision to be approved. When voting in a meeting, it is a simple majority of the voting members present.  When voting through email or other forms of asynchronous communication, it is a simple majority of all Technical Committee voting members.
+Simple majority is required for any decision to be approved. When voting in a meeting, it is a simple majority of the voting members present.  When voting through email or other forms of asynchronous communication, it is a simple majority of all Technical Committee voting members. For PRs in the community repository the same rule applies, simple majority of all Technical Committee voting members.
```

Comment:
Patching the previous sentence would've made the paragraph less repetitive, but this works of course.

> When voting through email or other forms of asynchronous communication, **including PRs in the community repository**, it is a simple majority of all Technical Committee voting members.